caddyhttp/proxy/reverseproxy.go

```diff
@@ -185,6 +186,8 @@ func NewSingleHostReverseProxy(target *url.URL, without string, keepalive int) *
 		rp.Transport = &http.Transport{
 			Dial: socketDial(target.String()),
 		}
+	} else if target.Scheme == "quic" {
+		rp.Transport = &h2quic.QuicRoundTripper{}
```

Comment:
Do you mean the `rp.Transport = &http.Transport{Dial: socketDial(target.String())}` ?
Or 
```
// if keepalive is equal to the default,
// just use default transport, to avoid creating
// a brand new transport
transport := &http.Transport{
	Proxy:                 http.ProxyFromEnvironment,
	Dial:                  defaultDialer.Dial,
	TLSHandshakeTimeout:   10 * time.Second,
	ExpectContinueTimeout: 1 * time.Second,
}
if keepalive == 0 {
	transport.DisableKeepAlives = true
} else {
	transport.MaxIdleConnsPerHost = keepalive
}
if httpserver.HTTP2 {
	http2.ConfigureTransport(transport)
}
rp.Transport = transport
```?
I think those options are not part of `h2quic.QuicRoundTripper`, so they cannot be set. But I'll take another look when I'll write those tests.